config/config-builder/src/validator_config.rs

```diff
@@ -41,6 +41,7 @@ pub struct ValidatorConfig {
     pub safety_rules_token: Option<String>,
     pub seed: [u8; 32],
     pub template: NodeConfig,
+    pub vault_network_timeout_ms: u64,
```

Comment:
I think we can just use the default, no need to expose this... how did we do the default for safety rules?

Reply:
We should use the default.  The question is if we want to hardcode the default value on line 226 or hardcode the default value on line 65.  I prefer hardcoding that value in `default()`, but leave that as a judgement call for you.